apps/web/src/components/blocksuite/workspace-header/styles.css.ts

```diff
@@ -149,7 +149,24 @@ export const pageListTitleWrapper = style({
   justifyContent: 'center',
   alignItems: 'center',
 });
-
+export const allPageListTitleWrapper = style({
+  fontSize: 'var(--affine-font-base)',
+  color: 'var(--affine-text-primary-color)',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  '::after': {
```

Comment:
Is this valid? I mean it is not inside of `selectors`

Reply:
I think it invalid

Reply:
it valid